topic-operator/src/main/java/io/strimzi/operator/topic/TopicOperatorMain.java

```diff
@@ -61,9 +61,10 @@ public class TopicOperatorMain implements Liveness, Readiness {
         Objects.requireNonNull(config.labelSelector());
         this.config = config;
         var selector = config.labelSelector().toMap();
-        this.kubernetesClient = new OperatorKubernetesClientBuilder("strimzi-topic-operator", TopicOperatorMain.class.getPackage().getImplementationVersion()).build();
+        this.kubernetesClient = new OperatorKubernetesClientBuilder("strimzi-topic-operator", 
+            TopicOperatorMain.class.getPackage().getImplementationVersion()).build();
```

Comment:
If you don't like it being on a single line because it is too long, you should probably not mix contructor on two lines with another call on one of them. E.g.

```suggestion
        this.kubernetesClient = new OperatorKubernetesClientBuilder("strimzi-topic-operator",
                TopicOperatorMain.class.getPackage().getImplementationVersion())
                .build();
```